.github/workflows/tests_uv.yaml

```diff
@@ -18,6 +18,12 @@ jobs:
         run: uv python install
         working-directory: lumigator/python/mzai/backend
 
+      - name: Start backend
+        run: make start-lumigator
```

Comment:
We should be forcing a build here so we're testing against the current version of the code. I think there's a way to specify both `build` and `image` in the docker-compose file, but that will require a bit more testing. Overall agree with using the non-mounted image though

Reply:
Hum, should we add a build as a previous step?

So far, it's been testing the current commit without exception, even if we're not mounting the volume.

Reply:
AFAICT it should work ok as it is right now?

Reply:
It is working as expected, yes. Let's keep an eye on it in case later we need to force a build later on.

Reply:
> It is working as expected, yes. Let's keep an eye on it in case later we need to force a build later on.
This is actually testing against a completely arbitrary version of lumigator due to https://mzlai.slack.com/archives/C07LJP9Q5KJ/p1729259395864809